sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -126,7 +126,9 @@ private[hive] class SparkExecuteStatementOperation(
 
   override def runInternal(): Unit = {
     setState(OperationState.PENDING)
-    logInfo(s"Submitting query '$redactedStatement' with $statementId")
+    logInfo(
+      log"Submitting query '${MDC(REDACTED_STATE, redactedStatement)}' with " +
```

Comment:
REDACTED_STATEMENT

Reply:
Ending with the abbreviation `STATE` looks a bit weird, it sounds like a `state`